tools/AzPreview/AzPreview.psd1

```diff
@@ -186,7 +186,7 @@ RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.10.4'; },
             @{ModuleName = 'Az.SqlVirtualMachine'; RequiredVersion = '1.1.0'; }, 
             @{ModuleName = 'Az.Ssh'; RequiredVersion = '0.1.0'; }, 
             @{ModuleName = 'Az.StackHCI'; RequiredVersion = '1.4.0'; }, 
-            @{ModuleName = 'Az.Storage'; RequiredVersion = '5.2.0'; }, 
+            @{ModuleName = 'Az.Storage'; RequiredVersion = '5.2.1'; },
```

Comment:
Technically, only the non-GA previews (<1.0.0) are included in Az preview. “-preview” are not. But since storage branch won’t be merged to main. It’s fine.